lib/uswds/rails/engine.rb

```diff
@@ -0,0 +1,18 @@
+module Uswds
+  module Rails
+    class Engine < ::Rails::Engine
+      initializer 'uswds-rails.assets.precompile' do |app|
+        %w(stylesheets javascripts fonts images).each do |sub|
```

Comment:
Looks like the Rails Engine documentation and generator puts all Rails engines in `lib/` -- see http://guides.rubyonrails.org/engines.html

How about this structure:

```
lib/  # our engine class
app/  # our override assets
vendor/  # uswds assets
spec/  # our tests
```

make sense? that seems to be what other assets-via-rails-gem projects do, and similar to what is in the Engine docs.